src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -32,7 +32,7 @@ public double getPos() {
   }
```

Comment:
changes to ClimberSubsystem.java should have been done in the climber PR

Reply:
I think it's fine to merge here.  They're just updates to the docs.

Reply:
at this point they may as well be merged here but this shows that we merged the climber PR before we were ready

Reply:
I don't think we should hold off on merging working code because docs aren't completely ready. I do think it would be cleaner to fix docs all at once in a single 'clean up' branch, but realistically that branch would never be made because it would constantly be out of date. If we want to prioritize good docs over a clean git history, I think doc changes should be made whenever we notice something is off.